Seasons If else

```diff
@@ -0,0 +1,32 @@
+public class Main {
+    public static void main(String[] args) {
+        int a = 1;
```

Comment:
It's not correct.
User should input the number from the console and we compare this input with the numbers like in your code below